crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1431,12 +1431,14 @@ impl World {
     /// The returned pointer must not be used to modify the resource, and must not be
     /// dereferenced after the immutable borrow of the [`World`] ends.
     ///
+    /// If the resource type is `!Send`, you must not share the returned pointer between threads.
```

Comment:
This is incorrect. `!Send` means the value cannot be sent to another thread. Sharing the pointer, `&T`, with other threads is required by `Sync`.

Reply:
ah, right